src/styles/card.scss

```diff
@@ -0,0 +1,137 @@
+.card {
+  display: flex;
+  align-items: center;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__outer {
+    display: flex;
+    flex-direction: column;
+    padding: 32px 16px 16px;
+  }
+
+  &__img-cover {
+    margin: auto;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin: 40px 0 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $product-name-font-color;
+    transition: all 300ms;
+
+    .card:hover & {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $product-code-font-color;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 8px;
+  }
+
+  &__view {
+    display: flex;
+    align-self: center;
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $product-review-font-color;
+  }
+
+  &__price-outer {
+    display: flex;
+    justify-content: space-between;
+    // margin-bottom: 16px;
```

Comment:
Remove the commented out code `// margin-bottom: 16px;` or uncomment it if it is necessary.